apps/snitch_core/lib/core/data/schema/taxanomy/taxonomy.ex

```diff
@@ -7,6 +7,7 @@ defmodule Snitch.Data.Schema.Taxonomy do
 
   schema "snitch_taxonomies" do
     field(:name, :string)
+    field(:taxons, :any, virtual: true)
```

Comment:
This field is used only when `get_all_taxonmy` is called. It would be nice perhaps, if `dump_taxonomy` took a taxonomy, loaded all the `taxons` in it and returned that -- to justify this field.